modules/server/src/test/scala/seqexec/server/TestCommon.scala

```diff
@@ -160,7 +164,8 @@ object TestCommon {
     Some("127.0.0.1"),
     0,
     3.seconds,
-    10.seconds
+    10.seconds,
+    32
```

Comment:
Is the amount of keywords coming from some logic or just a heuristic?

Reply:
Heuristics with a bit of real data. I tabulated the size of the keywords messages that failed with a timeout, and then chose 32 because I like powers of 2.